build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 
 group = 'org.bahmni.sms'
 version = '0.0.1'
-sourceCompatibility = '11.0.10'
+sourceCompatibility = '1.8'
```

Comment:
Try to rebuild the image again with `amazoncoretto:11.0.18`. Let's not downgrade :)-